From 7535a3526dfe78db02a08ca2fa6bf69f393105dd Mon Sep 17 00:00:00 2001
Message-Id: <7535a3526dfe78db02a08ca2fa6bf69f393105dd.1649664482.git.geliang.tang@suse.com>
From: Weqaar Janjua <weqaar.janjua@gmail.com>
Date: Thu, 10 Dec 2020 11:54:35 +0000
Subject: [PATCH] selftests/bpf: Xsk selftests - adding xdpxceiver to
 .gitignore
Git-commit: 7535a3526dfe78db02a08ca2fa6bf69f393105dd
Patch-mainline: v5.11-rc1
References: bsn#127
Modified-by-SEL: No


This patch adds *xdpxceiver* to selftests/bpf/.gitignore

Reported-by: Yonghong Song <yhs@fb.com>
Suggested-by: Yonghong Song <yhs@fb.com>
Signed-off-by: Weqaar Janjua <weqaar.a.janjua@intel.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20201210115435.3995-1-weqaar.a.janjua@intel.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/.gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/bpf/.gitignore b/tools/testing/selftests/bpf/.gitignore
index 752d8edddc66..f5b7ef93618c 100644
--- a/tools/testing/selftests/bpf/.gitignore
+++ b/tools/testing/selftests/bpf/.gitignore
@@ -36,3 +36,4 @@ test_cpp
 /runqslower
 /bench
 *.ko
+xdpxceiver
-- 
2.34.1

